### PR TITLE
fix(be): Job database is empty

### PR DESCRIPTION
**Pull request by nowfox**
**What is the current behavior?

Please describe the current behavior of the code that you are trying to modify. This could include any issues or limitations that you are addressing.

**What is the updated behavior?

Please describe the changes you are making to the code's behavior. This could include new features or enhancements that you are adding, as well as how they differ from the current behavior.

**Checklist

Please check off the following items before submitting your pull request:

- [ ] I have tested my changes.
- [ ] No sensitive information is included.
- [ ] I've reviewed my changes to ensure they won't cause any new issues that could affect the stability or performance of the codebase.

Please check off the following items if this changes include API modefications:

- [ ] I confirm this changes has been test with [authorization validation steps](https://github.com/awslabs/sensitive-data-protection-on-aws/blob/main/CONTRIBUTING.md#contributing-via-pull-requests), it won't break the authorization token verification mechanism.
- [ ] I confirm this changes won't cause security issues.

### CODE REVIEW

**Review by yanbasic — DISMISSED** [review]
LGTM

File: source/constructs/api/discovery_job/service.py [hunk 1]
```diff
@@ -203,6 +203,10 @@ def __start_run(job_id: int, run_id: int):
     job = crud.get_job(job_id)
     run = crud.get_run(run_id)
     run_databases = run.databases
+    if 0 == len(run_databases):
```
Comment (NingLu) [inline source/constructs/api/discovery_job/service.py:206]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)

Comment (nowfox) [reply source/constructs/api/discovery_job/service.py]: Modified

**Review by yanbasic — DISMISSED** [review]
LGTM

**Review by yanbasic — APPROVED** [review]
LGTM

**Pull request merged**